examples/api-call/manifest.json

```diff
@@ -0,0 +1,47 @@
+{
+  "version": "0.1.0",
+  "createdAt": "2022-07-26T12:51:00.000000Z",
+  "expiresAt": "2100-01-01T00:01:00.000000Z",
+  "metadata": {
+    "name": "External API call example",
+    "description": "Example manifest of a service making an outbound call to the external API",
+    "version": "0.1.0"
+  },
+  "payload": [
+    {
+      "platform": {
+        "arch": "x86_64",
+        "os": "linux"
+      },
+      "urls": [
+        "http://yacn2.dev.golem.network:8000/docker-golem-curl-jq-latest-9a163ce9d3.gvmi"
+      ],
+      "hash": "sha3:5f315c3756ec14a9dd4ed1a230ab4bbaabcc813d979e3f11cefd5458"
+    }
+  ],
+  "compManifest": {
+    "version": "0.1.0",
+    "script": {
+      "commands": [
+        "run .*",
```

Comment:
Maybe we should restrict this to allow only run `curl` using this image?
I'm not sure about this, but it would show better how someone can set restrictions in manifest

Reply:
Agree it is better to show it allows to restrict executed commands. Unfortunately from VM perspective `curl` is really `/bin/sh [...] curl [..]` so pattern took form of `run .*curl.*`. 
Also when pattern is not matched properly following exception is printed:
```
[2022-08-03T09:01:10.627+0200 ERROR yapapi.services.service_runner] Unhandled engine error                                                                                                                         
Traceback (most recent call last):                                                                                                                                                                                 
  File "/home/przemek/Golem/yapapi/yapapi/services/service_runner.py", line 287, in _run_instance                                                                                                                  
    fut_result = yield batch
  File "/home/przemek/Golem/yapapi/yapapi/engine.py", line 692, in process_batches
    remote = await activity.send(batch, deadline=batch_deadline)
  File "/home/przemek/Golem/yapapi/yapapi/rest/activity.py", line 86, in send
    batch_id = await self._api.call_exec(self._id, yaa.ExeScriptRequest(text=script_txt))
  File "/home/przemek/.cache/pypoetry/virtualenvs/yapapi-HO02pEWA-py3.10/lib/python3.10/site-packages/ya_activity/api_client.py", line 205, in __call_api
    raise e
  File "/home/przemek/.cache/pypoetry/virtualenvs/yapapi-HO02pEWA-py3.10/lib/python3.10/site-packages/ya_activity/api_client.py", line 193, in __call_api
    response_data = await self.request(
  File "/home/przemek/.cache/pypoetry/virtualenvs/yapapi-HO02pEWA-py3.10/lib/python3.10/site-packages/ya_activity/rest.py", line 268, in POST
    return await self.request(
  File "/home/przemek/.cache/pypoetry/virtualenvs/yapapi-HO02pEWA-py3.10/lib/python3.10/site-packages/ya_activity/rest.py", line 180, in request
    raise ApiException(http_resp=r)
ya_activity.exceptions.ApiException: (400)
Reason: Bad Request
HTTP response headers: <CIMultiDictProxy('content-length': '437', 'content-type': 'application/json', 'date': 'Wed, 03 Aug 2022 07:01:10 GMT')>
HTTP response body: {"message":"Bad request: Manifest violation in ExeScript: script validation error: no matching manifest entry found for 'run /bin/sh -c GOLEM_PRICE=`curl -X 'GET'                     'https://api.coingecko.com/api/v3/simple/price?ids=golem&vs_currencies=usd'                     -H 'accept: application/json' | jq .golem.usd`;                 echo ---;                 echo \"Golem price: $GOLEM_PRICE USD\";                 echo ---;'"}
```
I am not exactly familiar with Python, but probably there is a more graceful way to print this error message.